WeslToml.md

```diff
@@ -0,0 +1,118 @@
+# `wesl.toml` file format
+
+WESL shader projects have a standardized `wesl.toml` file, which describes the project setup. It is used by tools such as language servers and linkers[^1].
+
+The format looks as follows. As much of it is optional as possible.
+
+```toml
+[package]
+# Version of WESL used in the local shaders
+edition = "unstable_2025"
+
+# Where are the shaders located. This is the path of `package::`.
+# We watch this directory for changes.
+root = "./shaders"
+
+# Optional. Default value is all wesl and wgsl files in the root directory.
+include = [ "shaders/**/*.wesl", "shaders/**/*.wgsl" ]
```

Comment:
Add some documentation akin to https://www.typescriptlang.org/tsconfig/#include

That would make it clear that "absolute paths" and other such weird cases aren't supported.